### PR TITLE
Add proxy package for libxrandr

### PR DESCRIPTION
**Pull request by czoido**
Specify library name and version:  **libxrandr/system**

This is a proxy package for libxrandr. This means that this package will verify if xrandr is installed in the user's system and populate the cpp_info with the required information.
When, in the near future, a package for this library is created, this can be transparently substituted in the consumer without requiring any changes.

Related to: #641 #678 

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SSE4** [comment]
@czoido are we planning to add proxy package for each X11 library (there are around 50 of them...)?

File: recipes/libxrandr/all/conanfile.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                "This library is only compatible with Linux")
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def package(self):
+        tools.download("https://gitlab.freedesktop.org/xorg/lib/libxrandr/raw/master/COPYING", filename="COPYING")
+        self.copy("COPYING", dst="licenses")
+
+    def package_info(self):
+        if self._system_packages().installed("pkg-config"):
```
Comment (ericLemanissier) [inline recipes/libxrandr/all/conanfile.py]: You could use tools.which, to be even more distribution agnostic ?

Comment (SSE4) [reply recipes/libxrandr/all/conanfile.py]: yes, that would be the greatest. I'd prefer to stay as distribution agnostic as possible (and don't use `SystemPackageTools` interface at all)

**Comment by czoido** [comment]
> @czoido are we planning to add proxy package for each X11 library (there are around 50 of them...)?

I was planning just for: `libx11`, `libxrandr`, `libxinerama`, `libxcursor`, `libxi` to add `glfw` after they are merged.

**Comment by ericLemanissier** [comment]
I guess you plan to do it for libgl/mesa too ?

**Comment by czoido** [comment]
> I guess you plan to do it for libgl/mesa too ?

Yes, that's the idea

**Comment by czoido** [comment]
Closing: https://github.com/conan-io/conan-center-index/issues/641#issuecomment-575184483
Will open one for `xorg-dev` encompassing several libs.

**Pull request closed without merging**